### PR TITLE
Add the (initial) pet status detail page

### LINKED ISSUE #18: Design Pet Interaction Page
UI. This page will have: Pet Name, Pet Stats, Pet Picture, Train; Feed; and Race buttons

### PR DESCRIPTION
**Pull request by Chris-Johnston**
Fix #18 

Adds the page that shows the details about a pet status, and including all of the buttons used
to interact with the pet.

Adds 'dead.jpg' image, shown when a pet is dead.

### CODE REVIEW

**Comment by Chris-Johnston** [comment]
Ready for review, this should just perform the basic functionality for now

File: PetGame/Pages/Pet.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                {
+                    // pet not found, or wrong owner
+                    Response.StatusCode = 404;
+                    return;
+                }
+            }
+            else
+            {
+                // http unauthorized
+                Response.StatusCode = 404;
```
Comment (anishprasad01) [inline PetGame/Pages/Pet.cshtml.cs]: Unauthorized is 401

**Review by anishprasad01 — CHANGES_REQUESTED** [review]
Looks good, other than one small typo.

**Review by anishprasad01 — APPROVED** [review]

**Comment by anishprasad01** [comment]
LGTM. Approved

**Pull request merged**